internal/fs/fs.go

```diff
@@ -121,13 +121,16 @@ func NewFileSystem(
 
 	mtimeClock := timeutil.RealClock()
 
+	contentCache := contentcache.New(cfg.TempDir, mtimeClock)
+	contentCache.PopulateCache()
```

Comment:
What if the gcsfuse doesn't want a cache? Would this scan the `/tmp` unnecessarily?

Reply:
I will change this so "RecoverCache()" is only called when the user specifies the content cache flag.